ui/apps/platform/src/Containers/Vulnerabilities/ExceptionManagement/hooks/useRequestCVEsDetails.ts

```diff
@@ -0,0 +1,85 @@
+import { gql, useQuery } from '@apollo/client';
+import { getRequestQueryStringForSearchFilter } from 'utils/searchUtils';
+import { VulnerabilityException } from 'services/VulnerabilityExceptionService';
+import { getImageScopeSearchValue } from '../utils';
+import { sortCveDistroList } from 'Containers/Vulnerabilities/WorkloadCves/sortUtils';
+
+type AffectedImagesCountQueryResult = {
+    imageCVEs: {
+        cve: string;
+        affectedImageCount: number;
+        distroTuples: {
+            summary: string;
+            operatingSystem: string;
+            cvss: number;
+            scoreVersion: string;
+        }[];
+    }[];
+};
+
+export const affectedImagesCountQuery = gql`
+    query getAffectedImagesCount($query: String) {
+        imageCVEs(query: $query) {
+            cve
+            affectedImageCount
+            distroTuples {
+                summary
+                operatingSystem
+                cvss
+                scoreVersion
+            }
+        }
+    }
+`;
+
+type RequestCVEDetail = {
+    cve: string;
+    summary: string;
+    numAffectedImages: number;
+};
+
+type UseAffectedImagesCountResult = {
+    isLoading: boolean;
+    totalAffectedImageCount: number;
+    requestCVEsDetails: RequestCVEDetail[];
+};
+
+function useRequestCVEsDetails(exception: VulnerabilityException): UseAffectedImagesCountResult {
```

Comment:
I changed the `useAffectedImagesCount` hook to `useRequestCVEsDetails` because we want to fetch the extra information to display the CVE, summary, and the number of affected images in the `CVE selections` section of the modal